modules/commands.py

```diff
@@ -162,6 +174,12 @@ class Commands:
 		action=lambda debugger: debugger.clear_all_breakpoints(),
 		flags=Command.menu_widget|Command.menu_commands|Command.menu_main,
 	)
+	record = Command (
```

Comment:
I think this needs to be moved into gdp.py and include the `GDB:` prefix similar to how LLDB commands are done in lldb.py its not part of the dap spec.

Reply:
done, not sure if there is a better way to get/call `session.request`